processor/deltatocumulativeprocessor/README.md

```diff
@@ -0,0 +1,84 @@
+# Cumulative to Delta Processor
+
+| Status                   |                           |
+|--------------------------|---------------------------|
+| Stability                | [beta]                    |
```

Comment:
this is copied from cumulativetodeltaprocessor, I can change this to "under development"